lib/message.rb

```diff
@@ -9,7 +9,7 @@ def self.all
     end
 
     result = connection.exec('SELECT * FROM peeps')
```

Comment:
SELECT * FROM peeps ORDER BY id DESC - this short addition 'ORDER BY id DESC' will present the messages in descending order.